engine/engine_test.go

```diff
@@ -151,17 +151,19 @@ func (nowt BadTransport) RoundTrip(*http.Request) (*http.Response, error) {
 	return resp, nil
 }
 
+
 func TestNew(t *testing.T) {
 	os.Clearenv()
 	os.Setenv("AWS_CREDENTIAL_FILE", "/i/hope/this/path/does/not/exist")
 	defer os.Unsetenv("AWS_CREDENTIAL_FILE")
 
 	fixtures := []struct {
+		configfunc func(context.Context, ...func(*awsconfig.LoadOptions) error) (aws.Config,error)
```

Comment:
```suggestion
		configFunc func(context.Context, ...func(*awsconfig.LoadOptions) error) (aws.Config,error)
```
go prefers camel-case